theme.config.js

```diff
@@ -19,7 +19,7 @@ const TITLE_WITH_TRANSLATIONS = {
 export default {
   repository: 'https://github.com/vercel/swr',
   docsRepository: 'https://github.com/vercel/swr-site',
-  titleSuffix: ' – SWR',
+  titleSuffix: '', // It was left empty with the intention of avoiding duplicated title on homepage ---> SWR - SWR
```

Comment:
This is intentional, so on subpages it will be something like "Get Started — SWR" 🤔